proto/proto.go

```diff
@@ -0,0 +1,12 @@
+//go:build tools
+// +build tools
+
+package proto
+
+// https://github.com/grpc-ecosystem/grpc-gateway#installation
+import (
```

Comment:
Do we need to add this file to ensure developers have installed the gRPC dependencies?

Reply:
Probably not. This is only needed when people generate the proto, but this isn't used in our code.

We can add a README file to help users install these library. I can do that.